saltproc/simulation.py

```diff
@@ -140,13 +144,69 @@ def store_after_repr(self, after_mats, waste_dict, dep_step):
                     # Save isotope indexes map and units in EArray attributes
                     earr.flavor = 'python'
                     earr.attrs.iso_map = iso_idx
+
+                earr, iso_wt_frac = self._add_missing_nuclides(db, earr, iso_idx, iso_wt_frac)
```

Comment:
I don't love this naming convention, `earr`. It is super unclear to me what it represents.

Reply:
I don't like it either, but it's consistent with the current naming scheme in `simulation.py`. I will be fixing all the variable names in `simulation.py` in a future PR.